examples/ec2/availabilityzonegroup.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: ec2.aws.upbound.io/v1beta1
+kind: AvailabilityZoneGroup
+metadata:
+  annotations:
+    meta.upbound.io/example-id: ec2/v1beta1/availabilityzonegroup
+  labels:
+    testing.upbound.io/example-name: example
+  name: us-west-2-den-1
+spec:
+  forProvider:
+    optInStatus: opted-in
+    region: us-west-2
+
```

Comment:
Please clean these empty lines and separators from the examples. They cause build to fail with 

```
up: error: failed to build package: failed to parse examples: error converting YAML to JSON: yaml: line 2: did not find expected node content
```